docs/en/table_design/table_types/Comparison_of_abilities_of_various_tables.md

```diff
@@ -0,0 +1,43 @@
+# Comparison of abilities of various tables
+
+## Key columns and sort key
+
+<table>
+<thead>
+<tr><th width= "10"> </th><th><strong>Primary Key</strong> <strong>table</strong></th><th><strong>Duplicate Key table</strong></th><th><strong>Aggregate table</strong></th><th><strong>Unique Key table</strong></th></tr>
+</thead>
+<tbody><tr><td><strong>Key columns and UNIQUE constraints</strong> </td><td>The primary key has the UNIQUE constraint and NOT NULL constraint. </td><td>The duplicate key does not have the UNIQUE constraint. </td><td>The aggregate key has the UNIQUE constraint. </td><td>The unique key has the UNIQUE constraint. </td></tr><tr><td><strong>Logical relationships between key columns and data changes</strong> <strong></strong> </td><td>Insert, delete, and update operations are performed based on the primary key. If the primary key value of a new data row is same as that of an existing data row in the table,  unique constraint violation occurs. Then the new data row will replace the existing data row.<br />Compared to the Unique Key table, the Primary Key table has an enhanced underlying storage engine and therefore can replace the Unique Key table.</td><td>The duplicate key does not have the UNIQUE  constraint. So, if the Duplicate Key value of a new data row is same as that of an existing data row in the table, both the new and old data rows are retained in the table. </td><td>Insert, delete, and update operations are performed based on the aggregate key. So, if the Aggregate Key value of a new data row is same as that of an existing data row in the table, the new and old data rows will be aggregated based on the Aggregate Key and the aggregate functions of value columns. </td><td>Insert, delete, and update operations are performed based on the unique key.If the Unique Key value of a new data row is same as that of an existing data row in the table, the new data row will replace the existing data row.<br />The Unique Key table can be seen as an Aggregate table whose aggregate function is replace.</td></tr><tr><td><strong>Relationship between key columns and sort key</strong></td><td>Since v3.0, the key columns and sort key are decoupled. </td><td colspan="3">The key columns and sort key are coupled. </td></tr><tr><td><strong>Data types supported by key columns and sort key</strong>  </td><td>Numeric (including integers and BOOLEAN), string, and date (DATE and DATETIME). </td><td colspan="3">Numeric (including integers, BOOLEAN, and DECIMAL), string, and date (DATE and DATETIME).</td></tr><tr><td><strong>Relationship between key columns and partitioning/bucketing columns</strong> </td><td>Partitioning columns and bucketing columns must be in the primary key. </td><td>None </td><td>Partitioning columns and bucketing columns must be in the aggregate key. </td><td>Partitioning columns and bucketing columns must be in the unique key. </td></tr></tbody>
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8b6af35f-c05e-491b-91af-6bc0e809a175)
Use lower case for these types of keys